src/main/java/sunny/Task.java

```diff
@@ -0,0 +1,50 @@
+package sunny;
+
+/**
+ * Represents information of tasks specified by user
+ */
+public class Task {
+    // FIELDS-----------------------------
+    String description;
+    Boolean isDone;
+
+    /**
+     * Initialises description of tasks and set the status of completion
+     * @param description describe the task, ie eat lunch
+     * @param isDone if the tasks is completed
+     */
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Initialises description of task and set as not done
+     * @param description describe the task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void setter (boolean x) {
```

Comment:
It's better to use a verb for a method name. It can be a descriptive name like `setCompletionStatus`.